Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -164,6 +168,25 @@ QObject* AzAssetBrowserWindow::createListenerForShowAssetEditorEvent(QObject* pa
     return listener;
 }
 
+void AzAssetBrowserWindow::resizeEvent(QResizeEvent* resizeEvent)
+{
+    // leftLayout is the parent of the tableView
+    // rightLayout is the parent of the preview window.
+    // Workaround: When docking windows this event keeps holding the old size of the widgets instead of the new one
+    // but the resizeEvent holds the new size of the whole widget
+    // So we have to save the proportions somehow
+    const QWidget* leftLayout = m_ui->m_leftLayout;
+    const QVBoxLayout* rightLayout = m_ui->m_rightLayout;
+
+    const float oldLeftLayoutWidth = aznumeric_cast<float>(leftLayout->geometry().width());
+    const float oldWidth = aznumeric_cast<float>(leftLayout->geometry().width() + rightLayout->geometry().width());
+
+    const float newWidth = oldLeftLayoutWidth * aznumeric_cast<float>(resizeEvent->size().width()) / oldWidth;
+    
+    emit SizeChangedSignal(aznumeric_cast<int>(newWidth));
+    QWidget::resizeEvent(resizeEvent);
+}
+
```

Comment:
It doesn't look like the asset browser is using dock widgets internally for the left/right. This looks like it just needs a proper horizontal layout instead of using a splitter. We shouldn't need to do this hack.

Reply:
Discussed offline, I agree with @AMZN-daimini comment below that this change is good as-is, and we can save the layout refactor as a backlog item. @AMZN-Igarri has created a backlog issue for us to track this work:

https://github.com/o3de/o3de/issues/4259